data/templates/nginx/server.tpl.conf

```diff
@@ -93,3 +93,84 @@ server {
     access_log /var/log/nginx/{{ domain }}-access.log;
     error_log /var/log/nginx/{{ domain }}-error.log;
 }
+
```

Comment:
This file is a template changed, replicated and used by every domains of the instance.
Do we want the `xmpp-upload` domain to be specified multiple times?
I would suggest to include a kind of mechanism in YunoHost to always create this domain name. That’s it.

---
Thanks for what you are doing. I am looking forward to use this feature.

Reply:
As I said earlier, I really think that each yunohost domain should have his own XMPP components. I've created [issue #1516](https://github.com/YunoHost/issues/issues/1516 ) to discuss about this.